reference/forms/types/password.rst

```diff
@@ -76,4 +76,12 @@ The default value is ``''`` (the empty string).
 
 .. include:: /reference/forms/types/options/required.rst.inc
 
-.. include:: /reference/forms/types/options/trim.rst.inc
+trim
+~~~~
+
+**type**: ``Boolean`` **default**: ``false``
+
+If true, the whitespace of the submitted string value will be stripped
+via the ``trim()`` function when the data is bound. This guarantees that
```

Comment:
can you maybe use the ``:phpfunction:`trim` `` role? This will automatically generate a link to PHP.net.